python/llm/src/bigdl/llm/optimize.py

```diff
@@ -19,7 +19,7 @@
 from bigdl.llm.utils.common import invalidInputError
 
 
-def optimize_model(model, low_bit, optimize_llm=True):
+def optimize_model(model, low_bit='sym_int4', optimize_llm=False):
```

Comment:
what's the default value of `pretrained`? We should make it consistent

Reply:
I'll check with @yangw1234 about what is the preferred default value.

Reply:
just set it to true

Reply:
done